fmriprep/workflows/bold/fit.py

```diff
@@ -467,13 +468,21 @@ def init_bold_fit_wf(
             desc='coreg',
             name='ds_coreg_boldref_wf',
         )
+        ds_boldmask_wf = init_ds_boldmask_wf(
+            bids_root=layout.root,
+            output_dir=config.execution.fmriprep_dir,
+            desc='brain',
+            name='ds_boldmask_wf',
+        )
 
         # fmt:off
         workflow.connect([
             (hmcref_buffer, fmapref_buffer, [("boldref", "boldref_files")]),
             (fmapref_buffer, enhance_boldref_wf, [("out", "inputnode.in_file")]),
             (fmapref_buffer, ds_coreg_boldref_wf, [("out", "inputnode.source_files")]),
             (ds_coreg_boldref_wf, regref_buffer, [("outputnode.boldref", "boldref")]),
+            (fmapref_buffer, ds_boldmask_wf, [("out", "inputnode.source_files")]),
```

Comment:
Other workflows just set 

```
        ds_hmc_wf.inputs.inputnode.source_files = [bold_file]
```

I don't think I would worry about getting sbref vs bold straight here, as it's just a mask.